src/main/java/org/jenkinsci/test/acceptance/po/View.java

```diff
@@ -58,7 +58,7 @@ public void delete() {
         configure();
         clickLink("Delete View");
         waitFor(by.button("Yes"));
-        clickButton("Yes");
+        control(by.button("Yes")).clickAndWaitToBecomeStale();
```

Comment:
can this use submit?

Reply:
Wouldn't be exactly the same. In this form the 'Yes' button is not a submit button, which means that the click event is handled by some javascript function, which triggers submitting form.

Calling .submit() is the same as triggering a submit event on the form, so that javascript function won't be executed. If the javascript function is only submitting the form, it's the same, but if it's doing something else, we're causing a different behavior impossible to reproduce using only the UI